infrastructure/scripts/create-testdata/create-many-releases.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# create-many-releases.sh MIN MAX
+# Where MIN is the first release number used, and MAX is the last. Make sure that MAX>=MIN and MIN>=0.
+# This script creates a few releases, while in parallel we call undeploy.
+# This is useful to test the lock feature (db.lockType).
+# To check if the test is successful or not:
+# select applications from environments where name='development'; select appname from releases where appname='echo-???'
+# The test is successful if EACH application either appear in both tables, or in neither.
+
+cd "$(dirname "$0")" || exit 1
+
+set -u
+
+MIN=${1}
+MAX=${2}
+
+./create-environments.sh
+
+# Setup: ensure the app exists:
+for appId in $(seq -w "${MIN}" "${MAX}" ); do
+  RELEASE_VERSION=99999 ~/projects/fdc/kuberpult/infrastructure/scripts/create-testdata/create-release.sh "echo-${appId}";
```

Comment:
There shouldn't be absolute path here. For some other person it may not be in `project/fdc...`

Reply:
done